website/search/util.py

```diff
@@ -16,6 +16,8 @@
 
 RE_XML_ILLEGAL_COMPILED = re.compile(RE_XML_ILLEGAL)
 
+RE_HTML_TAG_COMPILED = re.compile(r'<[^>]+>')
```

Comment:
Woo thanks! Didn't see that there, very  useful. Was thinking this was a bit extreme!